server/src/modules/events/infra/repos/implementations/mikro-event-repo.ts

```diff
@@ -1,16 +1,17 @@
-import { QueryOrder } from '@mikro-orm/core'
-import { DB } from '../../../../../shared/infra/db'
-import { EventEntity } from '../../../../../shared/infra/db/entities/event.entity'
+import { EntityRepository, QueryOrder } from '@mikro-orm/core'
 import { Event } from '../../../domain/entities/event'
+import { EventEntity } from '../../../../../shared/infra/db/entities/event.entity'
 import { EventMap } from '../../../mappers/event-map'
 import { EventRepo } from '../event-repo'
 import { Result } from '../../../../../shared/core/result'
 import { AppError } from '../../../../../shared/core/app-error'
 
 export class MikroEventRepo implements EventRepo {
+  constructor(protected eventsEntityRepo: EntityRepository<EventEntity>) {}
+
   async getAllEvents(): Promise<Result<Array<Event>, AppError.UnexpectedError>> {
     try {
-      const eventEntities: Array<EventEntity> = await DB.eventsEntityRepo.find(
+      const eventEntities: Array<EventEntity> = await this.eventsEntityRepo.find(
         {},
```

Comment:
I've seen other codebases write a single line description of the intent of a db query so its easy for devs to understand. Maybe something we want to standardize in the future for more complicated db queries, not necessary here since it's conveyed by func name.

Reply:
Probably out of scope for this PR, but it's a good idea for the future. Comments rarely hurt, so we should try to have more of them in general